src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,8 @@
+.logo {
+
+  &__img {
+    display: flex;
```

Comment:
For a single image, using 'display: flex;' and 'justify-content: center;' is unnecessary. You may use 'margin: 0 auto;' instead to center the image if needed.